pom.xml

```diff
@@ -250,7 +250,7 @@
         <artifactId>apache-rat-plugin</artifactId>
         <version>${maven-rat-plugin.version}</version>
         <configuration>
-          <excludeSubprojects>false</excludeSubprojects>
+          <!-- <excludeSubprojects>false</excludeSubprojects> -->
```

Comment:
Just delete it?

Reply:
Doh. I forgot about this one. yes, I'll do a quick update to this PR to just delete it.  Thanks!